src/css/mapbox-gl.css

```diff
@@ -196,6 +196,12 @@ a.mapboxgl-ctrl-logo {
     background-image: svg-load('svg/mapboxgl-ctrl-logo.svg');
 }
 
+a.mapboxgl-ctrl-logo.mapboxgl-compact {
+    width: 21px;
+    height: 21px;
+    background-image: svg-load('svg/mapboxgl-ctrl-logo-compact.svg');
```

Comment:
Rather than adding another SVG, could we hide the wordmark portion of the existing logo SVG by toggling a CSS class?

Reply:
Nice idea but since the logo is loaded as a background-image I don't think it's possible to control that through standard CSS, I think it would need to be an embedded SVG for that to work :thinking:. Given it doesn't add much weight to the file size, I'd be tempted to leave it as is